fe/fe-core/src/main/java/com/starrocks/external/hive/HiveMetastoreApiConverter.java

```diff
@@ -52,6 +54,12 @@
 public class HiveMetastoreApiConverter {
 
     public static final IdGenerator<ConnectorTableId> CONNECTOR_ID_GENERATOR = ConnectorTableId.createGenerator();
+    private static final String SPARK_SQL_SOURCE_PROVIDER = "spark.sql.sources.provider";
+
+    private static boolean isDeltaLakeTable(Map<String, String> tableParams) {
```

Comment:
plz move `HudiTable.isHudiTable()` to here.

Reply:
Done